src/index.html

```diff
@@ -1,5 +1,16 @@
 <!doctype html>
+<div
 <html lang="en">
+<button onclick="location.reload()" class="btn btn-primary btn-lg" style="
```

Comment:
While this works and solves the current use case, this is not an ideal solution.
We will need to consider a generic way of `routing` between different components. Since this project is built on Angular, the best way will be to use Angular Routing. Read more here: https://angular.io/guide/router

Please implement Angular Routing instead.

Reply:
Hi Sir,
I will work on the solution that you have told. Angular framework is new for me .I am learning a lot of new things in this  project. Kindly give me some time I will send you the correct solution soon.
Thank you.

Reply:
Sure. Take your time. 
Since we already use angular I do not want to mix stuff.

You can ping me if you need help.